.deploy/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM rust:latest
 RUN apt-get -y update
 RUN apt-get install -y build-essential cmake pkg-config libssl-dev clang libclang-dev llvm
 
-ARG TOOLCHAIN=nightly-2021-01-25
+ARG TOOLCHAIN=nightly-2021-02-15
```

Comment:
is that the version that substrate is using? If not, does it make sense to use more recent nightly if we switch?

Reply:
This version seems stable enough for now, but I'm happy to upgrade to an even newer version if you want.

Reply:
I was thinking that we could upgrade the nightly every two months and then something like 03-15 might be nice. That way we could just plan this upgrade every fourth sprint or so. I'm kind of leaving it up to you what you prefer, but I guess it best if we update on a regular basis.